src/docfx/lib/HtmlUtility.cs

```diff
@@ -16,9 +16,7 @@ namespace Microsoft.Docs.Build
 {
     internal static class HtmlUtility
     {
-        public delegate void TransformHtmlDelegate(ref HtmlToken token);
-
-        private static readonly string[] s_allowedStyles = new[] { "text-align: right;", "text-align: left;", "text-align: center;" };
+        public delegate void TransformHtmlDelegate(ref HtmlReader reader, ref HtmlToken token);
```

Comment:
why is `HtmlToken` not `ref struct` just `struct`?

Reply:
`HtmlToken` and `HtmlAttributes` are bullets (think `JToken`), `HtmlReader`, `HtmlWriter` are guns (think `JsonSerializer`).